goose.py

```diff
@@ -263,21 +268,22 @@ def get_diff(db_migrations: List[Migration], file_system_migrations: List[Migrat
         return old_branch, new_branch
 
 
-def CINE_migrations_table(cursor) -> None:
+def CINE_migrations_table(cursor, migration_table) -> None:
```

Comment:
We are passing `migration_table` variable between different methods.
How about
* Creating a module level variable with default table name
* Override once if a table name is provided
* And use module level variable at all places?